src/TestLogger.bs

```diff
@@ -1,5 +1,5 @@
 namespace RBS_LOGGER
-class Logger
+class SLogger
```

Comment:
I'll remove this at the end... I need to handle the name collision in the codebase i'm testing these changes with.

Reply:
I think `RooibosLogger` is fine, unless this is a specific `SocketLogger`